### PR TITLE
Copy change for clarity

### PR DESCRIPTION
**Pull request by quinnkeast**
Tommy called out a confusing message due to ambiguous language. Changes "... can currently only handle searching **over** 50 repositories at a time" (could be interpreted as "can only search when more than 50 repositories are targeted") to "... can currently only handle searching **across** 50 repositories at a time".

### CODE REVIEW

**Review by christinaforney — APPROVED** [review]
Thanks for making this so quickly! Improves things, and we should also talk about if we want to keep the tracking issue linked here (and it should be a link not just text).

**Comment by rrhyne** [comment]
Tracking issues in UI isn't a pattern we use elsewhere. IMO, @quinnkeast's feedback text could be incorporated into all alerts to gather suggestions.

**Comment by quinnkeast** [comment]
I'd be in favour of removing the tracking link for now. It feels fragile and out of place, and isn't really a clear "step to take" in response to the message (there isn't one right now).

File: cmd/frontend/graphqlbackend/search_alert.go [hunk 1]
```diff
@@ -632,14 +632,14 @@ func alertForError(err error, inputs *SearchInputs) *searchAlert {
 		alert = &searchAlert{
 			prometheusType: "exceeded_diff_commit_search_limit",
 			title:          fmt.Sprintf("Too many matching repositories for %s search to handle", rErr.ResultType),
-			description:    fmt.Sprintf(`%s search can currently only handle searching over %d repositories at a time. Try using the "repo:" filter to narrow down which repositories to search, or using 'after:"1 week ago"'. Tracking issue: https://github.com/sourcegraph/sourcegraph/issues/6826`, strings.Title(rErr.ResultType), rErr.Max),
+			description:    fmt.Sprintf(`%s search can currently only handle searching across %d repositories at a time. Try using the "repo:" filter to narrow down which repositories to search, or using 'after:"1 week ago"'. Tracking issue: https://github.com/sourcegraph/sourcegraph/issues/6826`, strings.Title(rErr.ResultType), rErr.Max),
```
Comment (quinnkeast) [inline cmd/frontend/graphqlbackend/search_alert.go:635]:
```suggestion
			description:    fmt.Sprintf(`%s search can currently only handle searching across %d repositories at a time. Try using the "repo:" filter to narrow down which repositories to search, or using 'after:"1 week ago"'.`, strings.Title(rErr.ResultType), rErr.Max),
```

File: cmd/frontend/graphqlbackend/search_alert.go [hunk 2]
```diff
@@ -632,14 +632,14 @@ func alertForError(err error, inputs *SearchInputs) *searchAlert {
...
 		}
 		alert.priority = 2
 	} else if errors.As(err, &tErr) {
 		alert = &searchAlert{
 			prometheusType: "exceeded_diff_commit_with_time_search_limit",
 			title:          fmt.Sprintf("Too many matching repositories for %s search to handle", tErr.ResultType),
-			description:    fmt.Sprintf(`%s search can currently only handle searching over %d repositories at a time. Try using the "repo:" filter to narrow down which repositories to search. Tracking issue: https://github.com/sourcegraph/sourcegraph/issues/6826`, strings.Title(tErr.ResultType), tErr.Max),
+			description:    fmt.Sprintf(`%s search can currently only handle searching across %d repositories at a time. Try using the "repo:" filter to narrow down which repositories to search. Tracking issue: https://github.com/sourcegraph/sourcegraph/issues/6826`, strings.Title(tErr.ResultType), tErr.Max),
```
Comment (quinnkeast) [inline cmd/frontend/graphqlbackend/search_alert.go:642]:
```suggestion
			description:    fmt.Sprintf(`%s search can currently only handle searching across %d repositories at a time. Try using the "repo:" filter to narrow down which repositories to search.`, strings.Title(tErr.ResultType), tErr.Max),
```

File: cmd/frontend/graphqlbackend/search_alert_test.go [hunk 3]
```diff
@@ -165,12 +165,12 @@ func TestAlertForDiffCommitSearchLimits(t *testing.T) {
 		{
 			name:                 "diff_search_warns_on_repos_greater_than_search_limit",
 			multiErr:             multierror.Append(&multierror.Error{}, &RepoLimitError{ResultType: "diff", Max: 50}),
-			wantAlertDescription: `Diff search can currently only handle searching over 50 repositories at a time. Try using the "repo:" filter to narrow down which repositories to search, or using 'after:"1 week ago"'. Tracking issue: https://github.com/sourcegraph/sourcegraph/issues/6826`,
+			wantAlertDescription: `Diff search can currently only handle searching across 50 repositories at a time. Try using the "repo:" filter to narrow down which repositories to search, or using 'after:"1 week ago"'. Tracking issue: https://github.com/sourcegraph/sourcegraph/issues/6826`,
```
Comment (quinnkeast) [inline cmd/frontend/graphqlbackend/search_alert_test.go:168]:
```suggestion
			wantAlertDescription: `Diff search can currently only handle searching across 50 repositories at a time. Try using the "repo:" filter to narrow down which repositories to search, or using 'after:"1 week ago"'.`,
```

File: cmd/frontend/graphqlbackend/search_alert_test.go [hunk 4]
```diff
@@ -165,12 +165,12 @@ func TestAlertForDiffCommitSearchLimits(t *testing.T) {
...
 		},
 		{
 			name:                 "commit_search_warns_on_repos_greater_than_search_limit",
 			multiErr:             multierror.Append(&multierror.Error{}, &RepoLimitError{ResultType: "commit", Max: 50}),
-			wantAlertDescription: `Commit search can currently only handle searching over 50 repositories at a time. Try using the "repo:" filter to narrow down which repositories to search, or using 'after:"1 week ago"'. Tracking issue: https://github.com/sourcegraph/sourcegraph/issues/6826`,
+			wantAlertDescription: `Commit search can currently only handle searching across 50 repositories at a time. Try using the "repo:" filter to narrow down which repositories to search, or using 'after:"1 week ago"'. Tracking issue: https://github.com/sourcegraph/sourcegraph/issues/6826`,
```
Comment (quinnkeast) [inline cmd/frontend/graphqlbackend/search_alert_test.go:173]:
```suggestion
			wantAlertDescription: `Commit search can currently only handle searching across 50 repositories at a time. Try using the "repo:" filter to narrow down which repositories to search, or using 'after:"1 week ago"'.`,
```

**Review by rrhyne — APPROVED** [review]
LGTM

**Review by stefanhengl — APPROVED** [review]

**Pull request merged**